src/tests/functional/user/user-creation-many.test.ts

```diff
@@ -0,0 +1,82 @@
+import { describe, it, afterEach } from 'mocha';
+import { expect } from 'chai';
+import { prisma } from '../../test-setup';
+import { UserHelper } from '../../helpers/user-helper';
+import fs from 'node:fs';
+
+describe('Create users with csv suite (functional)', () => {
+  afterEach(async () => {
+    await prisma.user.deleteMany();
+  });
+
+  it('Test if createUsersWithCsv mutation can create many users', async () => {
```

Comment:
geralmente os testes inicial com `it('should...')`
com uma frase descritiva do que deve acontecer